src/Admin/ClarificationAdmin.php

```diff
@@ -30,12 +26,18 @@ class ClarificationAdmin extends AbstractAdmin
      *
      * @return Metadata
      */
-    public function getObjectMetadata($object)
+    public function getObjectMetadata(Clarification $object)
+        : Metadata
     {
-        return new Metadata($object->getTitle(), $object->getDescription(), $object->getMedia()->getPath());
+        return new Metadata(
```

Comment:
Why?

Reply:
PSR2 : Lines SHOULD NOT be longer than 80 characters; lines longer than that SHOULD be split into multiple subsequent lines of no more than 80 characters each.

Here the line lenght is over 80